workbench_access.post_update.php

```diff
@@ -96,3 +97,61 @@ function workbench_access_post_update_convert_user_storage_keys(array &$sandbox)
   $sandbox['#finished'] = empty($sandbox['ids']) ? 1 : ($sandbox['count'] - count($sandbox['ids'])) / $sandbox['count'];
   return t('Updated user assigments');
 }
+
```

Comment:
@larowlan I know this is a huge PR . Can you take a look at these two post_update hooks.

I think I have the correct, but I'm not entirely sure how the section keys are named in the current code.

Tests are passing, however.